scripts/release/create-canary-commands/build-artifacts.js

```diff
@@ -21,5 +21,5 @@ const run = async ({cwd, dry, tempDirectory}) => {
 };
 
 module.exports = async params => {
-  return logPromise(run(params), 'Building artifacts', 420000);
+  return logPromise(run(params), 'Building artifacts', 600000);
```

Comment:
Technically this change was unrelated but I noticed the previous default value was not very accurate, given the current average build time. (We've added more packages since the script was written.)